index.html

```diff
@@ -0,0 +1,134 @@
+          <li>/</li>
+          <li><a href="./blog.html">Blog</a></li>
+        </ul>
+      </nav>
+
+      <article class="main-content">
+        <div class="summary">
+          <img src="./images/home-intro.jpg" alt="A man hugging a golden dog">
+          <div>
+            <h2 id="main">Adopt A Dog</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam gravida nec ligula lobortis tincidunt. In faucibus nisi quis orci imperdiet, a suscipit odio tristique. Sed eu neque lectus. Phasellus faucibus in ex nec vestibulum. Cras ut sem vitae lacus bibendum volutpat et at ex. Fusce at massa tortor. Proin non nunc in felis vestibulum sodales. Nunc lobortis varius placerat. Etiam sed sapien eu mi semper blandit non eu justo. Morbi orci sapien, sollicitudin eget venenatis at, vehicula nec ipsum. Aliquam eget elit vitae turpis malesuada egestas non sit amet quam. In at rhoncus lorem. Mauris finibus velit sodales ligula luctus euismod. Curabitur in fringilla mauris. Nunc vestibulum massa eu mi maximus laoreet. Maecenas quis lobortis purus, vel vestibulum mauris.</p>
+          </div>
+        </div>
+
+        <div class="page-content">
+          <section>
+            <img src="./images/murphy-card.jpg" alt="A picture of Murphy, a brown and white dog" onclick="alert('This is ' + dogInfo('Murphy', 'a Boxer', 123.45))">
+            <h3>Murphy</h3>
+            <p><strong>Cost to Adopt:</strong> $123.45</p>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam gravida nec ligula lobortis tincidunt. In faucibus nisi quis orci imperdiet, a suscipit odio tristique.</p>
+            <button type="button" name="adopt" onclick="alert('Your total is: ' + totalCost(123.45))">Adopt</button>
+          </section>
+
+          <section>
+            <img src="./images/poppy-card.jpg" alt="A picture of Poppy, a white Shi Tsu" onclick="alert('This is ' + dogInfo('Poppy', 'a Shi Tsu', 123.45))">
+            <h3>Poppy</h3>
+            <p><strong>Cost to Adopt:</strong> $123.45</p>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam gravida nec ligula lobortis tincidunt. In faucibus nisi quis orci imperdiet, a suscipit odio tristique.</p>
+            <button type="button" name="adopt" onclick="alert('Your total is: ' + totalCost(123.45))">Adopt</button>
+          </section>
+
+          <section>
+            <img src="./images/jack-card.jpg" alt="A picture of Jack, a brown and white Beagle" onclick="alert('This is ' + dogInfo('Jack', 'a Beagle', 123.45))">
+            <h3>Jack</h3>
+            <p><strong>Cost to Adopt:</strong> $123.45</p>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam gravida nec ligula lobortis tincidunt. In faucibus nisi quis orci imperdiet, a suscipit odio tristique.</p>
+            <button type="button" name="adopt" onclick="alert('Your total is: ' + totalCost(123.45))">Adopt</button>
```

Comment:
Generally not a great practice to have two function calls in the onclick. You typically want to keep as little JS as possible in your HTML code. What would be better is to just have `totalCost(123.45)` and then make the alert in your function